homework.md

```diff
@@ -4,110 +4,109 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```

Comment:
I'm a bit confused about this change? What are the underscores?

Reply:
I have no idea how this happened, I'm certain I didn't do this, it makes no sense. I will try and correct the error.

Reply:
Oh, I wonder if you have a markdown auto-formatter? Maybe it corrects `*` to `_` because it took it as formatting to make words in italics, rather than a symbol?

Reply:
Yep, you were right. I narrowed it down to the Prettier code format extension. It kept seeing the question as ugly code and "auto-correcting" upon saving the file, so I disabled it and the problem was fixed.